src/ui/components/Scanner/Scanner.tsx

```diff
@@ -111,16 +116,30 @@ const Scanner = forwardRef(
     };
 
     const updateConnections = async (groupId: string) => {
-      // TODO: We should avoid calling getMultisigLinkedContacts every time we scan a QR code,
```

Comment:
I am not sure if I understand the new code below. Based on this comment we should avoid calling `getMultisigLinkedContacts`every time we scan a QR code, but I can see it's still there.

Reply:
I just cached connections base on `groupId` and use it when user scan QR with same groupId data. Am I correct ?

Reply:
Not sure about this tbh, maybe we ask @iFergal in case I misunderstood it.

Reply:
This feels off to me. If it's a group between Alice, Bob, Carol. The idea is if Alice has already scanned Bob and now scans Carol, that Carol's info just gets directly added to Redux rather than calling the core again.

This looks like it will use a cache that only has Bob in it.

I also don't know why we would need another Redux state `multiSigLinkContactsCache` when we already have `multisigGroupCache`. No need to put it in both places.

imo I would skip changing this TODO this close to alpha program - this is a risky change. It deserves it own PR at a later stage

Reply:
Ok so let's revert back this change and we will handle it on a separate ticket. I will bring it up during refinement tomorrow 👍

Reply:
Done. I revert it